esperanto/malkovro/fajro.md

```diff
@@ -0,0 +1 @@
+Vi metas la fajron proksime de la muro , kaj ĝi hardas . Ĝi odoris [familiara](http://thesugardivorce.com/wp-content/uploads/2013/01/Perfect-Burned-Marshmallows-on-a-stick.jpg) , kiel odoro de via infanaĝo ...
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.